server/src/main/java/org/elasticsearch/search/fetch/FetchContext.java

```diff
@@ -59,13 +56,6 @@ public ContextIndexSearcher searcher() {
         return searchContext.searcher();
     }
 
-    /**
```

Comment:
This is now moved to the wrapping SearchContext, so that inner hits can have their own version of the lookup that filters the source appropriately.